ocean_lib/ocean/mint.py

```diff
@@ -71,32 +66,14 @@ def deploy_fake_OCEAN():
         print("Need valid DEPLOYER_PRIVATE_KEY")
         sys.exit(0)
 
-    # ****DEPLOY****
     deployer_wallet = Wallet(web3, private_key=deployer_private_key)
-
-    print("****Deploy fake OCEAN: begin****")
-    # For simplicity, hijack DataTokenTemplate.
-    deployer_addr = deployer_wallet.address
-    OCEAN_cap = 1410 * 10 ** 6  # 1.41B
+    OCEAN_cap = 10000
     OCEAN_cap_base = util.to_base_18(float(OCEAN_cap))
-    OCEAN_token = DataToken(
-        DataToken.deploy(
-            web3,
-            deployer_wallet,
-            artifacts_path,
-            "Ocean",
-            "OCEAN",
-            deployer_addr,
-            OCEAN_cap_base,
-            "",
-            deployer_addr,
-        )
-    )
-    addresses["Ocean"] = OCEAN_token.address
-    print("****Deploy fake OCEAN: done****\n")
-
+    OCEAN_token = DataToken(address=network_addresses["development"]["Ocean"])
```

Comment:
All the other changes are about changing from "deploy & mint" to just "mint". But _this_ line is actually a _deploy_. I'm confused, what is the intent?

And I might have an answer. The idea of this issue overall is that we can _deprecate_ all of deploy & mint fake OCEAN here, because it's apparently already done in Barge. Therefore we should need any deploy(), or mint(), or really any of this code. Instead in the quickstart README we can simply add one line of code to mint() from within the demo code. _All_ of this can get deprecated.

Reply:
@trentmc here we are simply retrieving the token deployed by Barge. The deployment part was deprecated.

Reply:
Agreed with @calina-c. We can use instead the address for deployment from address.json as far as I understood.

Reply:
> @trentmc here we are simply retrieving the token deployed by Barge. The deployment part was deprecated.

Ah right.

But why do you need this script at all then? Why not just directly reference it in the unit tests etc?

Reply:
I think we could mint it on the spot, if necessary. We can discuss this together.